src/extensions.py

```
@@ -1,74 +1,107 @@
 
-    Returns:
-        Optional[User]: The User model instance if found; None if no session exists or the user is not found in the database.
+    try:
+        with Session(engine) as db:
+
+            session_id = request.cookies.get("session_id")
+            user = get_user_from_session_id(session_id, db)
+
+            if user:
+                db_routes = db.exec(
+                    select(Route).where(Route.user_id==user.id)
```

Ruff (0.16.0)

[error] Undefined name `Route`

(F821)